### ISSUE TITLE
random PU dcl identifier and wrongly assigned Cico No - please flag PU record

### ISSUE THREAD

**Issue by lightgivener** [pulibrary/cicognara-catalogo#121]
Hey @smallfind 

There was a mistake in the masterfile and PU records. Wrongly assigned Cico No and in PU records once again one of these weirdly random dcl identifiers that I can't find in the ShotList or anywhere else in my files.

https://catalog.princeton.edu/catalog/2068094 IS NOT Cico No 159 since it has a different title page and slightly different title: https://figgy.princeton.edu/catalog/8ce21cc1-9e3a-4ec5-9122-434c7b37cc01

It could be that it is a match for Cico No 161 but we don't have a fiche for that one and Cicognaras description is a bit complicated here.

Will have to get back to you on that one!

**Comment by smallfind** [comment]
Understood. Thanks! :-)

**Comment by lightgivener** [comment]
Sorry I am only seeing two of the comments on GitHub. Do you still need me to check something? Cheers

**Comment by smallfind** [comment]
I don't think so. Just waiting for your reply on this one? (https://github.com/pulibrary/cicognara-catalogo/issues/121)
